test/multest.jl

```diff
@@ -20,4 +20,10 @@ qv = QuArray(v)
 @assert qv'*qv == dot(v,v)
 @assert rawcoeffs(qv*qv') == v*v'
 
-@assert qv'*qm*qv == first(v'*m*v)
\ No newline at end of file
+@assert qv'*qm*qv == first(v'*m*v)
+
+@assert rawcoeffs(2*qv) == 2*v
+@assert rawcoeffs(qv/2) == v/2
+@assert rawcoeffs(scale(2,qv)) == 2*v
+@assert norm(qv) == norm(v)
+#@assert rawcoeffs(normalize(qv)) == rawcoeffs(qv)/norm(v)
```

Comment:
You might want to use `@test_approx_eq` or `@test_approx_eq_eps` here (see docs for [Base.Test](http://julia.readthedocs.org/en/release-0.3/stdlib/test/)).